app/actions.py

```diff
@@ -775,29 +777,29 @@ def findPrior(self, searchFor):
     self.find(searchFor, -1)
 
   def indent(self):
+    indentation = app.prefs.prefs['editor'].get('indentation')
+    indentationLength = len(indentation)
     if self.selectionMode == app.selectable.kSelectionNone:
-      self.cursorMoveAndMark(0, -self.penCol,
-          self.penRow - self.markerRow, self.penCol - self.markerCol, 0)
-      self.redo()
-      self.indentLines()
-    elif self.selectionMode == app.selectable.kSelectionAll:
-      self.cursorMoveAndMark(len(self.lines) - 1 - self.penRow, -self.penCol,
-          -self.markerRow, -self.markerCol,
-          app.selectable.kSelectionLine - self.selectionMode)
-      self.redo()
-      self.indentLines()
+      self.verticalInsert(self.penRow, self.penRow, self.penCol, indentation)
```

Comment:
Yeah it seemed a lot cleaner and since I made indent lines only for indenting the beginning of all lines (when you tab with a a non-none selection mode), I just used vertical insert. Both functions use verticalInsert and verticalInsert wasn't really used for anything yet so I figured it was okay.